deps/compose.template

```diff
@@ -22,14 +22,14 @@ services:
     command: -c "sleep 5 && ./mock_verifier_contract.sh {{ ETH_PRIV_KEY }}"
 
   bootstrapper_l1:
-    image: {{ IMAGE_REPOSITORY }}bootstrapper:latest
+    image: {{ IMAGE_REPOSITORY }}bootstrapper:1.1.0
```

Comment:
Adding the tag will break the execution when you try to run with local images

Reply:
maybe this can help:

```suggestion
    image: {{ IMAGE_REPOSITORY }}bootstrapper:1.1.0
    build:
       dockerfile: bootstrapper/Dockerfile
```

So when you call `docker-compose build` it should build the image using the Dockerfile in `dockerfile` and tag it with the same tag as in `image`